.travis.yml

```diff
@@ -33,6 +33,7 @@ env:
     - CONDA_CHANNEL_PRIORITY=True
     - NUMPY_VERSION=stable
     - INSTALL_HOLE="true"
+    - DUECREDIT_ENABLE=yes
```

Comment:
Try it with global env var here.

Then try it with setting
```python
os.environ["DUECREDIT_ENABLE"] = "yes"
```
in [MDAnalysisTests/__init__.py](https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/__init__.py) (and *not* in travis.yml).